pwndbg/next.py

```diff
@@ -103,6 +108,18 @@ def break_next_call(symbol_regex=None):
         if ins.symbol and re.match('%s$' % symbol_regex, ins.symbol):
             return ins
 
+
+def break_next_ret(address=None):
+    while pwndbg.proc.alive:
+        ins = break_next_branch(address)
+
+        if not ins:
+            break
+
+        if capstone.CS_GRP_RET in ins.groups:
```

Comment:
This error shouldn't occur on `ni` or `si`, I've debugged ARM stuff plenty before without this issue.